config/jobs/kubernetes/sig-cloud-provider/azure/sig-azure-config.yaml

```diff
@@ -269,3 +269,40 @@ periodics:
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: Runs conformance & node conformance tests latest kubernetes master with aks-engine
     testgrid-num-columns-recent: '30'
+- interval: 24h
+  name: ci-kubernetes-e2e-capz-conformance-master
+  decorate: true
+  labels:
+    preset-service-account: "true"
+    preset-azure-cred: "true"
+    preset-dind-enabled: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes-sigs
+    repo: cluster-api-provider-azure
+    base_ref: master
```

Comment:
using master because https://github.com/kubernetes-sigs/cluster-api-provider-azure/commit/ab3fa86ea3208382416d3cf909d56900a8d44293 is not in the latest release but planning on moving to a release tag once it is included in the next release.